src/java/org/apache/nutch/plugin/Plugin.java

```diff
@@ -88,9 +88,4 @@ public PluginDescriptor getDescriptor() {
   private void setDescriptor(PluginDescriptor descriptor) {
     fDescriptor = descriptor;
   }
-
-  @Override
-  protected void finalize() throws Throwable {
-    shutDown();
```

Comment:
I'm not sure we can simply remove the call to shutdown. I need to further investigate options and confirm.